lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -142,7 +142,8 @@ private enum OnVisibleAction {
   FontAssetDelegate fontAssetDelegate;
   @Nullable
   TextDelegate textDelegate;
-  private boolean enableMergePaths;
+  private LottieFeatureFlags lottieFeatureFlags = LottieFeatureFlags.getInstance();
```

Comment:
I agree that this should be on a per-composition basis. However, I think we should create a new instance of the flags rather than having a singleton because you may want a different set of flags on different drawables.

We'll also need to mirror this API for compose but I can help with that once we've settled on something.